Simplified/Announcements/NYPLAnnouncementBusinessLogic.swift

```diff
@@ -2,8 +2,12 @@ import Foundation
 
 private let announcementsFilename: String = "NYPLPresentedAnnouncementsList"
 
+protocol NYPLAnnouncementProvider: class {
```

Comment:
do we need this protocol? it seems unneeded since it's only used inside NYPLAnnouncementBusinessLogic... is it maybe for future unit tests?

Reply:
Forgot to remove that, thanks for the catch